src/js/components/List/List.js

```diff
@@ -105,32 +148,74 @@ const List = React.forwardRef(
     const Container = paginate ? StyledContainer : Fragment;
     const containterProps = paginate ? { ...theme.list.container } : undefined;
 
+    const [orderingData, setOrderingData] = useState();
+
+    const draggingRef = useRef();
+    
+    const ariaProps = {
+      role: onClickItem || onOrder ? 'listbox' : 'list',
+    };
+    
+    
+    if (active >= 0) {
+      let activeId;
+      // We have an item that is 'focused' within the list. This could
+      // be the list item or one of the up/down ordering buttons.
+      // We need to figure out an id of the thing that will be shown as active
+      if (onOrder) {
+        // figure out which arrow button will be the active one.
+        const btnId = (active % 2) ? 'MoveDown' : 'MoveUp';
```

Comment:
grommet tends to avoid compressing variable names, suggest 'btnId' => 'buttonId'.